haskell/test/Test/Ligo/BaseDAO/Token/Transfer.hs

```diff
@@ -110,34 +103,11 @@ transferTests = testGroup "Transfer:"
             transfer 1 unfrozenTokens nonexistent owner1 dao
               & expectCustomError #fA2_INSUFFICIENT_BALANCE dao (#required .! 1, #present .! 0)
 
-      , nettestScenarioCaps "cannot transfer frozen tokens" $ do
-          ((owner1, _), (owner2, _), dao, _, _) <- originateWithCustomToken
-
-          withSender (AddressResolved owner1) $
-            transfer 10 frozenTokens owner1 owner2 dao
-              & expectCustomErrorNoArg #fROZEN_TOKEN_NOT_TRANSFERABLE dao
```

Comment:
I think both this and the one removed above should be kept given the other comments.